settings.py

```diff
@@ -62,6 +62,7 @@
     ('fra', 'French'),  # we need this alias
     ('hin', 'Hindi'),
     ('sw', 'Swahili'),
+    ('por', 'Portuguese'),
```

Comment:
are there more languages that we have available on Transifex but not on HQ or this was the only one?